README.md

```diff
@@ -0,0 +1,29 @@
+
+## Synopsis
+Set of tools to snapshot & analyse procfs of a unix machine.
+
+ - shapshot.py: a tool to snapshot statistics from a (remote) unix machine.
+ - webserver.py: webserver useful for result analysis.
+
+## Installation
+Currently this project does not require installation.
+Required depencencies:
+ - sshpas
+ - twisted
+
+On a Debian they can be intalled calling:
+```
+  sudo apt-get install sshpass python-twisted

```

Comment:
s/intalled/installed/.

Worth noting that they can also be installed with `pip install -r requirements.txt`, which python devs would be used to.